### PR TITLE
[WIP][Feature] Update RTMW models

### PR DESCRIPTION
**Pull request by Tau-J**
## Checklist

**Before PR**:

- [ ] I have read and followed the workflow indicated in the [CONTRIBUTING.md](https://github.com/open-mmlab/mmpose/blob/master/.github/CONTRIBUTING.md) to create this PR.
- [ ] Pre-commit or linting tools indicated in [CONTRIBUTING.md](https://github.com/open-mmlab/mmpose/blob/master/.github/CONTRIBUTING.md) are used to fix the potential lint issues.
- [ ] Bug fixes are covered by unit tests, the case that causes the bug should be added in the unit tests.
- [ ] New functionalities are covered by complete unit tests. If not, please add more unit tests to ensure correctness.
- [ ] The documentation has been modified accordingly, including docstring or example tutorials.

**After PR**:

- [ ] CLA has been signed and all committers have signed the CLA in this PR.

### CODE REVIEW

File: mmpose/models/losses/classification_loss.py [hunk 1]
```diff
@@ -190,9 +204,15 @@ def forward(self, pred_simcc, gt_simcc, target_weight):
             pred = pred.reshape(-1, pred.size(-1))
             target = target.reshape(-1, target.size(-1))
 
-            loss += self.criterion(pred, target).mul(weight).sum()
+            t_loss = self.criterion(pred, target).mul(weight)
+
+            if self.mask is not None:
+                t_loss = t_loss.reshape(N, K)
+                t_loss[:, self.mask] = t_loss[:, self.mask] * self.mask
```
Comment (Ben-Louis) [inline mmpose/models/losses/classification_loss.py:211]: Should the last `self.mask` be replaced with `self.mask_weight`?

File: mmpose/configs/wholebody_2d_keypoint/rtmpose/cocktail13/rtmw-l_8xb320-270e_cocktail14-384x288.py [hunk 2]
```diff
@@ -0,0 +1,637 @@
...
+        in_featuremap_size=tuple([s // 32 for s in input_size]),
+        simcc_split_ratio=codec['simcc_split_ratio'],
+        final_layer_kernel_size=7,
+        gau_cfg=dict(
+            hidden_dims=256,
+            s=128,
+            expansion_factor=2,
+            dropout_rate=0.,
+            drop_path=0.,
+            act_fn=SiLU,
```
Comment (Ben-Louis) [inline mmpose/configs/wholebody_2d_keypoint/rtmpose/cocktail13/rtmw-l_8xb320-270e_cocktail14-384x288.py:131]: Should this `SiLU` be replaced with string `'SiLU'`?

File: mmpose/models/losses/classification_loss.py (hunk 1 above)
Comment (Tau-J) [reply mmpose/models/losses/classification_loss.py:211]: You're right. I'll fix it soon.
